src/Summoner/Template.hs

```diff
@@ -139,17 +174,22 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
         $endLine
         |]
 
-    createCabalTest :: Text
-    createCabalTest =
+    createCabalTest :: Text -> Text
+    createCabalTest r =
         [text|
-        test-suite ${repo}-test
+        test-suite             ${repo}-test
           type:                exitcode-stdio-1.0
           hs-source-dirs:      test
           main-is:             Spec.hs
           build-depends:       $base
                              , $repo
+                             $r
```

Comment:
You don't need `, $repo` if you have `$r`, The idea is that if there's no `library` stanza in project, we don't want to add dependency for this library.